_posts/README.md

```diff
@@ -2,44 +2,75 @@
+
+Once your post is ready to publish, we'll convert the Google Doc into Markdown for you (or you can do it yourself if you'd like), and open a pull request to [18f.gsa.gov][5] to stage the post for publication. 
+
+That's it! We may ask you a few questions or have some minor edits, but your job is done! You can stop reading here if you want.
+
+The rest of this document has instructions for creating the markdown file and pointers on how to format the HTML.
+
+**Most important bullet points:**
+
+* Blog posts need to be written in Markdown before publishing.
+  * Tools like [Pandoc][6] and [Gdocs2md][7] might help convert between file types
+* Pull requests should be issued to the `staging` branch from another branch
+  * All images embedded in posts or referenced in metadata need to be **added to the repository**, i.e., don't [hot link][8] to 3rd-party assets.
+* Each post needs some 18F-specific metadata in the front-matter: author names, tags, a short description, etc. (see below)
+* We deploy automatically through pull requests. That means **every pull request to production gets two sets of eyes**, where one of them is someone on the 18f.gsa.gov team other than the author. No exceptions: if it's time sensitive, start IMing or texting people.
+* We're an AP writing style shop. The grammar-inclined among us will try to review your work before live, but definitely ask the internet about proper AP solutions if you run up against a "one space after a period or two" sort of question. (The answer is one space, not two. :wink:)
 
 ### Creating a new blog post file
 
-Put draft posts that are ready for publication review in [`_posts/`](_posts). Use a filename that matches the example below. There should be a publication date and a unique slug.
+There is a script called `post` avilable that will create a post for you with all the appropriate metadata fields available. The [code lives here][9], but you can use it from the root of the website like this:
+
+```zsh
+script/post -t "The Encasement Strategy on Legacy Systems"
+```
+
+The `-t` or `--title` flags are the only required field and at its simplest will create a post with that title in the `_posts` directory with today's date as the publish date.
 
-For example, this filename:
+You can also use any of these options to add more information to that post:
 
 ```
-_posts/2014-09-08-the-encasement-strategy-on-legacy-systems-and-the.md
+    -d, --date DATE                  Date published (default, today)
+    -i, --image IMAGE                Post image
+    -a, --author AUTHOR              Post author (comma separated)
+    -g, --tags TAGS                  Tags for this post (comma separated)
+    -c, --content CONTENT            A markdown file containing the post's text
```

Comment:
Hoping someday it can accept a Google Docs URL, convert it to markdown and build the post. Someday.